app/views/provider/admin/dashboards/_ajax_load.html.slim

```diff
@@ -1,11 +1,6 @@
 javascript:
   document.addEventListener('DOMContentLoaded', function () {
     let widgetPath = #{json widget.path}
-    let serviceId = #{json widget.params[:service_id]}
 
-    if (serviceId === null) {
-      dashboardWidget.loadAudienceWidget(widgetPath)
-    } else {
-      dashboardWidget.loadServiceWidget(serviceId, widgetPath)
-    }
+    dashboardWidget.loadAudienceWidget(widgetPath)
```

Comment:
Not super necessary, but we could extract this from the partial and execute it in the pack js file.